cmd/metrics.go

```diff
@@ -280,6 +280,26 @@ func cacheMetricsPrometheus(ch chan<- prometheus.Metric) {
 			float64(cdStats.UsageState),
 			cdStats.Dir,
 		)
+
+		ch <- prometheus.MustNewConstMetric(
+			prometheus.NewDesc(
+				prometheus.BuildFQName("cache", "usage", "size"),
+				"Indicates current cache usage in bytes",
+				[]string{"disk"}, nil),
+			prometheus.GaugeValue,
+			float64(cdStats.UsageSize),
```

Comment:
Should this be an atomic load as well?

Reply:
disk usage/capacity is locally computed and there is no need to use atomic store/load - removed refs